q2_longitudinal/tests/test_longitudinal.py

```diff
@@ -362,71 +329,41 @@ def test_linear_mixed_effects_taxa(self):
         pdt.assert_frame_equal(obs, exp)
 
     def test_volatility(self):
+        # Just a simple "does it run?" test. Not much worth testing in terms
+        # of the rendered output - vega does all the heavy lifting for us.
         volatility(
             output_dir=self.temp_dir.name, metadata=self.md_ecam_fp,
-            metric='observed_otus', group_column='delivery',
-            state_column='month', individual_id_column='studyid',
-            spaghetti='yes')
-
-    def test_volatility_no_spaghetti_no_control(self):
-        volatility(
-            output_dir=self.temp_dir.name, metadata=self.md_ecam_fp,
-            metric='observed_otus', group_column='delivery',
-            state_column='month', individual_id_column='studyid',
-            spaghetti='no', plot_control_limits=False)
-
-    def test_volatility_mean_spaghetti(self):
-        volatility(
-            output_dir=self.temp_dir.name, metadata=self.md_ecam_fp,
-            metric='observed_otus', group_column='delivery',
-            state_column='month', individual_id_column='studyid',
-            spaghetti='mean')
-
-    def test_pointplot_from_dataframe(self):
-        plot_this = pd.DataFrame(
-            {'time': [0, 1, 2], 'metric': [2, 3, 4]}, index=['a', 'b', 'c'])
-        _pointplot_from_dataframe(
-            'time', 'metric', plot_this, None, ci=95, palette='Set1', ax=None,
-            legend=True, color=None, xtick_interval=None)
-
-    def test_volatility_table_data(self):
```

Comment:
since table inputs are currently breaking, maybe we should add a test for this? I appreciate that testing visualizations is a PITA so maybe don't test at the action level, just test that feature data is actually being merged and preserved in the metadata, such that it should be accessible in a qzv?

Reply:
an easy test might be to just use a feature ID as a `default_metric` since the validation comes after merging. Unless if features are being dropped later, e.g., by vega.